prototypes/technologies/tech.lua

```diff
@@ -217,7 +217,7 @@ data:extend{
         mod = "Tycoon",
         icon = "__tycoon__/graphics/technology/residential-housing.png",
         icon_size = 256,
-        prerequisites = { "steel-processing", "optics", "tycoon-meat-processing", "tycoon-baking", "tycoon-milking", "tycoon-bottling" },
+        prerequisites = { "steel-processing", "tycoon-meat-processing", "tycoon-baking", "tycoon-milking", "tycoon-bottling" },
```

Comment:
Was optics renamed/dropped in vanilla Factorio, or is there another reason for removing it here?

Reply:
dropped

Reply:
it is be merged to lamp